cdap-docs/user-guide/source/tutorials/logistic.rst

```diff
@@ -0,0 +1,237 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2017 Cask Data, Inc.
+    :description: The CDAP User Guide: Getting Started
+
+:hide-toc: true
+
+===============================================================
+Example: Building a Fraud Classification Machine Learning Model
+===============================================================
+
+Introduction
+------------
+In this tutorial, you will learn how to use CDAP to build and operationalize an enterprise-ready Machine Learning for prediciting credit fraud using only CDAP’s point-and-click interfaces.
+
+Scenario
+---------
+You are processing a large number of credit card transactions. Identfiying which transactions are fraudelent is business critical.
+
+- You want to train a Logistic Regression model using labeled data
+
+- You want to be able to send events to the model in realtime, and see which are predicted to be fraudelent
+
+Data
+----
+Click below to donwload a `.csv` file containing the data necessary to complete the tutorial.
+
+:download:`fraud_train.csv </_include/tutorials/fraud_train.csv>`
+
+Video Tutorial
+--------------
+
+..  youtube:: DyahfonSTRY
+
+Step-by-Step Walkthrough
+------------------------
+
+Loading the Data
+~~~~~~~~~~~~~~~~
+Download the data linked in the Data section above. Open Data Preparation, and upload `fraud_train.csv` as a "File." This is the data we will use to train our model.
+
+Cleaning and Exploring the Data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+First, you need to parse the data. From the `body` column drop-down menu, select Parse > CSV, setting the first row as the header. Next, from the `body` column drop-down, select Delete Column. You may also delete the `id` column, since we will not be using it in our regression. 
+
+We have eight predictive variables, all of which have been normalized except for Amount:
+
+- `normalized_total_spent_last_24_hours`: Total spent in the last 24 hours.
+- `normalized_merchant_fraud_risk`: Merchant's track record of fraudulent transactions.
+- `normalized_time_since_last_transaction`: Time since the last purchase.
+- `normalized_average_transaction`: Average transaction size.
+- `normalized_time_till_expiration`: Time until the card expires.
+- `normalized_daily_average_transaction`: Average amount transacted per day.
```

Comment:
`Average number of transaction per day`? Thought this was simpler. Feel free to ignore if you think otherwise.